setup.py

```diff
@@ -313,11 +313,13 @@ def get_pow_chain_head() -> PowBlock:
     pass
 
 
-def execution_state_transition(execution_state: ExecutionState, execution_payload: ExecutionPayload) -> None:
+def execution_state_transition(execution_state_root: Bytes32,
```

Comment:
This needs to be updated for the linter/build

Reply:
I fixed it.

added `return True` instead of pass in setup.py because of usage. What do you think?

Reply:
Makes sense to me since we have no tests for `process_execution_payload` yet